### PR TITLE
MueLu: Support for Repartitioning in Maxwell1

### PR DESCRIPTION
**Pull request by csiefer2**
Issue: #10407
Tests: New one added
Stakeholder feedback: Via email

### CODE REVIEW

**Review by cgcgcg — DISMISSED** [review]

**Comment by csiefer2** [comment]
@trilinos/framework Cuda tests aren't posting

**Comment by ZUUL42** [comment]
@csiefer2 they are currently doing to a different CDash server, trilinos-cdash.sandia.gov
You can find a query link under the dropdown arrow of the cuda build, for example:
```
>>> CDash URL1 = https://trilinos-cdash.sandia.gov/index.php?project=Trilinos&display=project&filtercount=3&showfilters=1&filtercombine=and&field1=site&compare1=61&value1=vortex32&field2=buildname&compare2=61&value2=PR-10408-test-ats2_cuda-10.1.243-gnu-8.3.1-spmpi-rolling_release_static_Volta70_Power9_no-asan_no-complex_no-fpic_mpi_pt_no-rdc_uvm_deprecated-on_no-package-enables-703&field3=buildstamp&compare3=61&value3=20220405-1626-Pull%20Request
```

**Comment by csiefer2** [comment]
Random Krino failure.  Ugh.

File: packages/muelu/adapters/xpetra/MueLu_Maxwell1_def.hpp [hunk 1]
```diff
@@ -361,14 +361,23 @@ namespace MueLu {
...
+        precList11_.set("repartition: use subcommunicators in place",true);
+      }        
+
         
     }
     else
       precList11_.remove("repartition: enable", false);
    
 
+    std::cout<<"**** CMS: precList22_ ****"<<std::endl <<precList22_ << std::endl;
```
Comment (cgcgcg) [inline packages/muelu/adapters/xpetra/MueLu_Maxwell1_def.hpp:380]: nope

File: packages/muelu/adapters/xpetra/MueLu_Maxwell1_def.hpp [hunk 2]
```diff
@@ -426,6 +435,8 @@ namespace MueLu {
         precList11_.set("coarse: type", "none");
       }
 
+      std::cout<<"**** CMS: precList11_ ****"<<std::endl <<precList11_ << std::endl;
```
Comment (cgcgcg) [inline packages/muelu/adapters/xpetra/MueLu_Maxwell1_def.hpp:438]: and nope :-P

**Review by cgcgcg — CHANGES_REQUESTED** [review]

**Review by cgcgcg — APPROVED** [review]

**Pull request merged**